in_toto/runlib_test.go

```diff
@@ -360,6 +360,23 @@ func TestRunCommand(t *testing.T) {
 	}
 }
 
+func TestRunCommandErrors(t *testing.T) {
```

Comment:
Can / should we also test by invoking `InTotoRun`?

Reply:
Oh whoops, I missed your comment on the PR.

Reply:
yes we can. I will add a few more tests for this.

Reply:
Reviving this one. I see what you mean but I think it's okay to test that the no command flag works as intended and still results in a link, no?

Reply:
@shibumi added a test checking if link metadata is successfully generated. Also caught that a minor difference in `byProducts` had crept in.